### PR TITLE
54Task-get all tasks

### PR DESCRIPTION
**Pull request by CezarN88**

### CODE REVIEW

File: src/main/java/com/teamrocket/tms/controllers/UserController.java [hunk 1]
```diff
@@ -61,4 +62,9 @@ public ResponseEntity<ProjectDTO> createProject(@PathVariable Long userId, @Vali
     public ResponseEntity<TeamDTO> createTeam(@PathVariable Long userId, @Valid @RequestBody TeamDTO teamDTO) {
         return ResponseEntity.ok(userService.createTeam(userId, teamDTO));
     }
-}
+    @GetMapping("/{userId}/tasks")
+    public ResponseEntity<List<TaskDTO>> getAllTasksForUser(@PathVariable Long userId) {
+        List<TaskDTO> tasks = userService.getAllTasksForUser(userId);
```
Comment (Daniel7Ionescu) [inline src/main/java/com/teamrocket/tms/controllers/UserController.java:67]: Please consider using the project standard for returns

**Review by Daniel7Ionescu — CHANGES_REQUESTED** [review]
the thing about the return

**Review by GabriellaNuca2209 — APPROVED** [review]
Good job, brother

**Review by Daniel7Ionescu — APPROVED** [review]
looks good to me

**Pull request merged**